source/cloud/cloud-administration/common-sso-google.rst

```diff
@@ -0,0 +1,62 @@
+Migrating from OAuth 2.0 to OpenID Connect
+------------------------------------------
+
+OAuth 2.0 is being deprecated and replaced by OpenID Connect. Refer to the product documentation to `convert your existing OAuth service provider configuration <https://docs.mattermost.com/cloud/cloud-administration/converting-oauth-2.0-to-openid-connect.html>`__ for Google Apps to the OpenID Connect standard.
+
+Configuring Google Apps as a Single Sign-On (SSO) service
+---------------------------------------------------------
+
+Follow these steps to configure Mattermost to use Google as a Single Sign-on (SSO) service for team creation, account creation, and sign-in.
+
+.. note::
+
+  The `Google People API <https://developers.google.com/people>`__ has replaced the Google+ API, which was deprecated by Google as of March 7th, 2019 `per their notice <https://developers.google.com/+/api-shutdown>`__.
+
+Step 1: Create OpenID Connect project in Google API Manager
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Go to `Google Cloud Platform <https://console.developers.google.com>`__.
```

Comment:
In the other document there were no line breaks between items. 0/5 as it doesn't affect the reader experience. :)